webservices/tasks/__init__.py

```diff
@@ -12,19 +13,25 @@
     schedule = {
         'refresh': {
             'task': 'webservices.tasks.refresh.refresh',
-            'schedule': crontab(minute=0, hour=9),
+            'schedule': crontab(minute=0, hour=9), 
+        },
+        
+        'reload_all_aos': {
+            'task': 'webservices.tasks.legal_docs.reload_all_aos',
+            'schedule': crontab(minute=15, hour=1),
```

Comment:
What is this schedule?

Reply:
this is when we plan to run reload of all AOs task daily.